integrations/ipfy_test.go

```diff
@@ -0,0 +1,71 @@
+// +build integration
+
+package integrations_test
+
+import (
+	. "friendly-potato/integrations"
+	"net"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestIPFy_GetExtIP(t *testing.T) {
+
+	t.Logf("setup ipfy")
+	f, err := NewIPFy()
+	if err != nil {
+		t.Logf("NewIPFy() failed: %v",err)
+		t.FailNow()
+	}
+
+	tests := []struct {
+		name    string
+		fields  *IPFy
+		wantIp  net.IP
```

Comment:
Codacy found an issue: [struct field wantIp should be wantIP](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)